guide/src/components/MarkdownElement.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import marked from 'marked';
+
+
+const styles = {
+  root: {
+    marginTop: 20,
+    marginBottom: 20,
+    padding: '0 10px',
+  },
+};
```

Comment:
This file is 2-space indented; which would imply that we added this file using an editor that isn't configured to use the [.editorConfig](https://github.com/cyverse/cyverse-ui/blob/master/.editorconfig#L6) file within the repository.